pom.xml

```diff
@@ -147,7 +147,7 @@
     If you changes codahale.metrics.version, you also need to change
     the link to metrics.dropwizard.io in docs/monitoring.md.
```

Comment:
The address corresponding to `metrics.dropwizard.io`  has not been changed, so this pr has not been changed `docs/monitoring.md`

Reply:
thank @sarutak